src/zx/FunctionalityConstruction.cpp

```diff
@@ -397,4 +417,21 @@ namespace zx {
         return false;
     }
 
+    PiExpression FunctionalityConstruction::parseParam(const qc::Operation* op,
+                                                       std::size_t          i) {
```

Comment:
```suggestion
                                                       const std::size_t          i) {
```

Reply:
A younger me would probably just write `int` in this case. But `const std::size_t` just shows you how beautiful C++ can be :upside_down_face: .

Reply:
🙃 it certainly is...